### PR TITLE
Remove stale text about documentation being available in English

### PR DESCRIPTION
**Pull request by aj11anuj**
What's the need to write that line "You're lucky, that documents are available in English" in any other language other than English and later at bottom translating it to English in small letters. 

Most of the people who are reading this docs/course are English readers and a very few people who know that language will only be able to grasp that line.

And if someone is viewing this course/docs in mobile, than that line at bottom in small letters is not noticable at first sight. When i saw this slide in first sight then i used Google Translate to translate that line and later i noticed it's translation is written in smaller letters at bottom.
I think it will be better to simply write that line in English only, and keep docs as precise as possible.

Page to update:
https://www.kubernetes.dev/docs/onboarding/09-documentation/

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
These pages were originally slides and aimed at an English-speaking audience.

The line “You are lucky that the documents are available in English!” in plain English is too blunt and making this change would send the wrong impression - that the Kubernetes project sincerely believes that statement.

We could use a Markdown footnote or other mechanism - possibly even a [shortcode](https://gohugo.io/content-management/shortcodes/) - to help people read the page and discover that a translation is available.

**Comment by mrbobbytables** [comment]
Honestly - I'd just remove the statement altogether. I don't think it would have made it through initial review had it been known what it stated

**Comment by jberkus** [comment]
Yeah, this is a joke about the word Kubernetes being Greek.  It was a fine joke for a presentation, but it doesn't work as docs.

**Comment by sftim** [comment]
/retitle Remove stale text about documentation being available in English

**Comment by sftim** [comment]
@aj11anuj please edit the issue description to tell people that you're commenting on https://www.kubernetes.dev/docs/onboarding/09-documentation/

It's otherwise really not obvious.

**Comment by sftim** [comment]
Please see https://github.com/kubernetes/contributor-site/pull/428#issuecomment-1704271344

Best to drop the line altogether, Greek and English.
